src/osm_configurator/model/project/calculation/aggregation_phase.py

```diff
@@ -21,9 +51,75 @@ def calculate(self, configuration_manager):
         After the calculations are done, the results are stored on the hard-drive.
 
         Args:
-            configuration_manager (configuration_manager.ConfigurationManager): The object containing all the configuration needed for execution.
+            configuration_manager_o (configuration_manager.ConfigurationManager): The object containing all the configuration needed for execution.
 
         Returns:
             calculation_state_enum.CalculationState: The state of the calculation, after this phase finished its execution or failed trying so.
         """
-        pass
+        # Get path to the results of the last Phase
+        checkpoint_folder_path_last_phase: Path = calculation_phase_utility.get_checkpoints_folder_path_from_phase(
+            configuration_manager_o,
+            calculation_phase_enum_i.CalculationPhase.GEO_DATA_PHASE)
+
+        # Get path to the results of the current Phase
+        checkpoint_folder_path_current_phase: Path = calculation_phase_utility.get_checkpoints_folder_path_from_phase(
+            configuration_manager_o,
+            calculation_phase_enum_i.CalculationPhase.TAG_FILTER_PHASE)
+
+        # Prepare result folder
+        deleter: FileDeletion = file_deletion_i.FileDeletion()
+        deleter.reset_folder(checkpoint_folder_path_current_phase)
+
+        # check if the folder exist
+        if checkpoint_folder_path_last_phase.exists() and checkpoint_folder_path_current_phase.exists():
+            list_of_traffic_cell_checkpoints: List = list(checkpoint_folder_path_last_phase.iterdir())
+        else:
+            return calculation_state_enum_i.CalculationState.ERROR_PROJECT_NOT_SET_UP_CORRECTLY, ""
```

Comment:
Were is this error catched? Because this is a situation that could happen in a normal use of the application and I donn't think the application should crash here

Reply:
fixed